src/index.js.flow

```diff
@@ -13,11 +13,10 @@ import type {
   MutationUpdaterFn,
 } from 'apollo-client';
 import type { DocumentNode, VariableDefinitionNode } from 'graphql';
-import gql from 'graphql-tag';
 
-// export everything from apollo-client
-export * from 'apollo-client';
-export { gql };
+export type NetworkStatus = 1 | 2 | 3 | 4 | 6 | 7 | 8;
+
+export var compose: $Compose;
```

Comment:
We will need to remove this.  `master` no longer exports `compose`.  See the current changelog.

Reply:
Thanks! I removed this from the commit.